cypress/e2e/login.cy.tsx

```diff
@@ -1,36 +1,34 @@
 const buttonText = 'Conectar con una clave secreta';
 const textLiLoginConfirm =
   'Copiar y pegar su clave secreta lo hace vulnerable a accidentes, ataques y estafas que pueden provocar la pérdida de fondos.';
-const secretKey = 'SDMKSLMZTXCXG4OAJVZKISPJ4NML2H23DWNXZI4XJBG3L2PNQ4XJYRJ5';
+const secretKey = 'SDMKSLMZTXCXG4OAJVZKISPJ4NML2H23DWNXZI4XJBG3L2PNQ4XJYRJ';
 describe('Button Component', () => {
   beforeEach(() => {
     cy.visit('/');
-    cy.contains('button', buttonText).should('be.visible');
     cy.contains('button', buttonText).click();
     cy.get('.check__input').as('checkboxInput');
+    cy.contains('li', textLiLoginConfirm).as('warning');
```

Comment:
Avoid specifying the HTML element in the variable because if we change the`li` and we use a div tomorrow, we've to change the variable name.